src/TcoCore/src/XaeTcoCore/XaeTcoCore.tsproj

```diff
@@ -11,11 +11,8 @@
 			</ArrayInfo>
 		</DataType>
 	</DataTypes>
-	<Project ProjectGUID="{39D84538-69DF-46F4-9E2F-272A81B72C66}" TargetNetId="172.20.10.2.1.1" Target64Bit="true" ShowHideConfigurations="#x106">
+	<Project ProjectGUID="{39D84538-69DF-46F4-9E2F-272A81B72C66}" TargetNetId="39.151.186.254.1.1" Target64Bit="true" ShowHideConfigurations="#x106">
```

Comment:
As far as I know each developer has to do it manually themselves, since it involves making changes to the .git folder. I can have a look if there is a better automatic way.